### PR TITLE
Spelling changelog

### PR DESCRIPTION
**Pull request by jsoref**
This fixes some misspellings in the Swift changelog, it is split per https://github.com/apple/swift/pull/42421#issuecomment-1101092698

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8994,7 +8994,7 @@ Swift 1.0
 [SE-0107]: <https://github.com/apple/swift-evolution/blob/main/proposals/0107-unsaferawpointer.md>
 [SE-0108]: <https://github.com/apple/swift-evolution/blob/main/proposals/0108-remove-assoctype-inference.md>
 [SE-0109]: <https://github.com/apple/swift-evolution/blob/main/proposals/0109-remove-boolean.md>
-[SE-0110]: <https://github.com/apple/swift-evolution/blob/main/proposals/0110-distingish-single-tuple-arg.md>
+[SE-0110]: <https://github.com/apple/swift-evolution/blob/main/proposals/0110-distinguish-single-tuple-arg.md>
```
Comment (jsoref) [inline CHANGELOG.md:8997]: This requires renaming the file in the other repository.

Comment (CodaFi) [reply CHANGELOG.md:8997]: Please send a PR here https://github.com/apple/swift-evolution/pulls

Comment (jsoref) [reply CHANGELOG.md:8997]: Sorry about the mass closures. I have some tooling that I use, but I accidentally copied the wrong repository name 🤦 , so it deleted all of my existing branches which caused github to close my PRs.

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Pull request merged**